src/sfast/triton/torch_ops.py

```diff
@@ -119,8 +119,10 @@ def forward(ctx, input, num_groups, weight=None, bias=None, eps=1e-05):
                     weight = weight.contiguous()
                 if bias is not None:
                     bias = bias.contiguous()
+                N, C = input.shape[:2]
+                HxW = input.numel() // (N * C)
                 output, mean, rstd = aten.native_group_norm(
-                    input, num_groups, weight, bias, eps)
+                    input, weight, bias, N, C, HxW, num_groups, eps)
```

Comment:
Hmm I'm getting an error as though this change was not made. i.e. num_groups is still passed at pos 2.

Reply:
I guess this didn't make it into release

Reply:
Yes, not released yet.😂 Building a dynamo is not easy.

Reply:
Yep, just checked that only the nightly release has this fix.